pkg/components/exporter/exporter.go

```diff
@@ -74,16 +76,33 @@ func NewDaemonSet(detail components.Detail, k *v1alpha1.KeplerInternal) *appsv1.
 	}
 
 	deployment := k.Spec.Exporter.Deployment.ExporterDeploymentSpec
-	image := k.Spec.Exporter.Deployment.Image
 	nodeSelector := deployment.NodeSelector
 	tolerations := deployment.Tolerations
-	port := deployment.Port
 	// NOTE: since 2 or more KeplerInternals can be deployed to the same namespace,
 	// we need to make sure that the pod selector of each of the DaemonSet
 	// create of each kepler is unique. Thus the daemonset name is added as
 	// label to the pod
 
-	bindAddress := "0.0.0.0:" + strconv.Itoa(int(port))
+	exporterContainer := newExporterContainer(k.Name, k.DaemonsetName(), k.Spec.Exporter.Deployment)
+
+	var containers []corev1.Container
+
+	var volumes = []corev1.Volume{
+		k8s.VolumeFromHost("lib-modules", "/lib/modules"),
+		k8s.VolumeFromHost("tracing", "/sys"),
+		k8s.VolumeFromHost("proc", "/proc"),
+		k8s.VolumeFromHost("kernel-src", "/usr/src/kernels"),
+		k8s.VolumeFromHost("kernel-debug", "/sys/kernel/debug"),
```

Comment:
we need a better way to handling volumes. - In another PR

 E.g. each `New<X>Container` can return `[]NamedMount`. 

```
type NamedMount string

const (
 HostLibModulesMount NamedMount = "host-lib-modules"
 HostProc  = "host-proc"
 KeplerConfigMapMount = "cm-kepler"
)

func (m HostMount) Volume() corev1.Volume {
   mounts := map[HostMount]string{
       LibModulesMount:  "/lib/modules", 
...
    }
    if strings.StartsWith("host-", m) {
     return k8s.VolumeFromHost(m, mounts[m])
   } else if strings.StartsWith("cm-", m)
     return k8s.VolumeFromConfigMap(m, mounts[m])
  }
 
}
```